### PR TITLE
Refs #18524 - Allow republishing of CVV repos

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: test/functional/content_view/version/republish_repositories_test.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require_relative '../../test_helper'
+require 'hammer_cli_katello/content_view_puppet_module'
+
+module HammerCLIKatello
+  describe ContentViewPuppetModule do
+    it 'allows republishing repositories on  a content view version' do
+      ex = api_expects(:organizations, :index) do |p|
```
Comment (akofink) [inline test/functional/content_view/version/republish_repositories_test.rb]: You can use the OrganizationHelpers https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/organization/organization_helpers.rb

File: lib/hammer_cli_katello/content_view_version.rb [hunk 2]
```diff
@@ -62,6 +62,19 @@ class InfoCommand < HammerCLIKatello::InfoCommand
...
     end
 
+    class RepublishRepositoriesCommand < HammerCLIKatello::SingleResourceCommand
+      include HammerCLIForemanTasks::Async
+
+      action :republish_repositories
+      command_name "republish-repositories"
+
+      success_message _("Content view is being deleted with task %{id}")
+      failure_message _("Could not delete the content view")
```
Comment (akofink) [inline lib/hammer_cli_katello/content_view_version.rb]: Are these really the right messages?

File: test/functional/content_view/version/republish_repositories_test.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+      ex = api_expects(:content_view_versions, :index) do |p|
+        p['version'] == '1' && p['content_view_id'] == 3
+      end
+      ex.returns(index_response([{'id' => 6}]))
+
+      api_expects(:content_view_versions, :republish_repositories) do |p|
+        p['id'] == 6
+      end
+
```
Comment (akofink) [inline test/functional/content_view/version/republish_repositories_test.rb]: Is there some ForemanTask being monitored/returned?

File: lib/hammer_cli_katello/content_view_version.rb (hunk 2 above)
Comment (jlsherrill) [inline lib/hammer_cli_katello/content_view_version.rb]: Copy paste fail!

**Pull request merged**